core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -197,7 +197,6 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
 
   test("BlockManagerId.isDriver() backwards-compatibility with legacy driver ids (SPARK-6716)") {
```

Comment:
Please update the test name as from now on it does not test "backwards-compatibility with legacy driver ids".

Reply:
@attilapiros 
Changed the testcase name, Thanks for your comment.